storage/cql_iface/tests/cas.py

```diff
@@ -0,0 +1,56 @@
+import atexit
```

Comment:
rename file to something more meaningful? :) Like: cassandra_cluster_manager.py or similar?